gamedata/tf2.cattr_starterpack.txt

```diff
@@ -897,7 +898,7 @@
 				// unuqie x-ref "damage_blast_push"
 				"library"	"server"
 				"linux"		"@_ZN9CTFPlayer19ApplyPushFromDamageERK15CTakeDamageInfo6Vector"
-				"windows"	"\x55\x8B\xEC\xA1\x2A\x2A\x2A\x2A\x83\xEC\x28\x83\x78\x30\x00\x53"
+				"windows"	"\x2A\x2A\x2A\x2A\x2A\x2A\x2A\x2A\x83\xEC\x28\x83\x78\x30\x00\x53\x56\x57\x8B\xF9"
```

Comment:
Guessing this is to enable other plugins to detour on this, which is fine.

Reply:
Correct. The signature of the original file is still correct, but I changed it on my end because my knockback modifier detours the same function. You're free to not put this change, I'll just change the signature on my plugin so there are no issues.